packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,13 +9,15 @@ and this project adheres to
 
 ## Unreleased
 
+## 3.0.0 - 2020-08-24
+
 ### Added
 
 - [#291](https://github.com/JupiterOne/sdk/issues/291) - Introduce
   `j1-integration document` command that is used to automatically generate
   documentation in `{integration-proj-dir}/docs/jupiterone.md`.
 
-- Fixed [#284](https://github.com/JupiterOne/sdk/issues/284) - Introduce
+- [#284](https://github.com/JupiterOne/sdk/issues/284) - Introduce
```

Comment:
I've started linking every change in the CHANGELOG to the issue. This is just a minor change for style consistency.